chains/evm/cli/account/import-priv-key.go

```diff
@@ -25,6 +33,12 @@ func BindImportPrivKeyFlags(cmd *cobra.Command) {
 func init() {
 	BindImportPrivKeyFlags(importPrivKeyCmd)
 }
+func ValidateImportPrivKeyFlags(cmd *cobra.Command, args []string) error {
+	if len(PrivateKey) != 64 {
+		return fmt.Errorf("invalid private key: length")
+	}
+	return nil
+}
```

Comment:
_Some_ validation for PK length already exists within the `secp256k1` package though perhaps it is better to present user with a message before it even gets that far